nucypher/blockchain/eth/sol/source/contracts/staking_contracts/StakingInterface.sol

```diff
@@ -144,17 +181,17 @@ contract StakingInterface {
     /**
     * @notice Mint tokens in the staking escrow
     */
-    function mint() external {
+    function mint() public {
         getStateContract().escrow().mint();
         emit Mined(msg.sender);
     }
 
     /**
     * @notice Withdraw available reward from the policy manager to the staking contract
     */
-    function withdrawPolicyReward(address payable _recipient) public {
-        uint256 value = getStateContract().policyManager().withdraw(_recipient);
-        emit PolicyRewardWithdrawn(_recipient, value);
+    function withdrawPolicyReward() public {
+        uint256 value = getStateContract().policyManager().withdraw();
+        emit PolicyRewardWithdrawn(msg.sender, value);
```

Comment:
To make possible to control eth flows, otherwise owner will withdraw everything. But with this changes - eth will go to staking contract and it means each staking control must decide who can withdraw and how much

Reply:
Ah yes, it makes a lot of sense. Nothing to see here then :)